dashboards-observability/public/components/application_analytics/components/config_components/service_config.tsx

```diff
@@ -29,6 +29,7 @@ interface ServiceConfigProps extends AppAnalyticsComponentDeps {
   dslService: DSLService;
   selectedServices: OptionType[];
   setSelectedServices: (services: OptionType[]) => void;
+  page?: string;
```

Comment:
Page is an optional parameter here but I don't see any null handling in the ServiceMap component?